lib/grpc/telemetry.ex

```diff
@@ -115,7 +115,8 @@ defmodule GRPC.Telemetry do
 
     :telemetry.span(@client_rpc, start_metadata, fn ->
       try do
-        {span_fn.(), start_metadata}
+        result = span_fn.()
+        {result, Map.put(start_metadata, :result, result)}
```

Comment:
Could we add this new `:result` metadata key to the moduledoc in the `Client-side Events -> Metadata` section? Similar to how it is `Server-side Events -> Metadata` section. 😄

Reply:
Yep good call @Nezteb